### PR TITLE
Add kpt command for package edit

### PR DESCRIPTION
**Pull request by mortent**
Add a kpt cli command to edit packages. Ref https://github.com/GoogleContainerTools/kpt/pull/3060

### CODE REVIEW

File: internal/cmdrpkgedit/command.go [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+	return newRunner(ctx, rcg).Command
+}
+
+func newRunner(ctx context.Context, rcg *genericclioptions.ConfigFlags) *runner {
+	r := &runner{
+		ctx: ctx,
+		cfg: rcg,
+	}
+	c := &cobra.Command{
+		Use:     "edit SOURCE_PACKAGE NAME",
```
Comment (martinmaly) [inline internal/cmdrpkgedit/command.go:66]: consider using the same identifiers here and line 33

File: internal/cmdrpkgedit/command.go [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+func (r *runner) preRunE(cmd *cobra.Command, args []string) error {
+	const op errors.Op = command + ".preRunE"
+	client, err := porch.CreateClient(r.cfg)
+	if err != nil {
+		return errors.E(op, err)
+	}
+	r.client = client
+
+	if len(args) < 2 {
+		return errors.E(op, fmt.Errorf("SOURCE_PACKAGE and NAME are required positional arguments; %d provided", len(args)))
```
Comment (martinmaly) [inline internal/cmdrpkgedit/command.go:104]: depending on which symbolic name is used for the 2nd positional parameter, this may need to be updated too

**Review by martinmaly — APPROVED** [review]
LGTM with couple of nits. let's add a test (https://github.com/GoogleContainerTools/kpt/tree/main/e2e/testdata/porch)

UPDATE_GOLDEN_FILES=1 will be helpful (that way one needs to only write the commands and the stdin/stdout are populated, modulo manual confirmation of correctness)

File: internal/cmdrpkgedit/command.go (hunk 1 above)
Comment (mortent) [reply internal/cmdrpkgedit/command.go:66]: Fixed.

**Pull request merged**